cobigen-cli/src/main/java/com/devonfw/cobigen/cli/commands/CobiGenCommand.java

```diff
@@ -8,7 +8,7 @@
  * This class defines the main CobiGen command
  */
 @Command(description = MessagesConstants.WELCOME_MESSAGE, name = "cobigen", aliases = { "cg" },
-    mixinStandardHelpOptions = true, version = "CobiGen CLI 1.0", subcommands = { GenerateCommand.class })
+    mixinStandardHelpOptions = true, version = {"CobiGen CLI 1.0","Java plug-in 2.1.0","OpenAPI plug-in	2.3.0","HTML plug-in 2.0.1"}, subcommands = { GenerateCommand.class })
```

Comment:
This is related to issue #956, right? Actually that is not enough, please read again the issue.

What we need to do is read the artificial pom, in order to parse the different versions of the plug-ins and add them there. If you have any question regarding this, contact me.